cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -170,6 +168,53 @@ function create_resource_from_string() {
   return 1;
 }
 
+function create_reconcile_resource_from_string() {
+  local -r config_string=$1;
+  local -r config_name=$2;
+  local -r namespace=$3;
+
+  # kubectl_output must be declared ahead of time to allow capturing kubectl's exit code and not local's exit code.
+  local kubectl_output;
+  # shellcheck disable=SC2086
+  # Disabling because "${KUBECTL_OPTS}" needs to allow for expansion here
+  kubectl_output=$(echo "${config_string}" | ${KUBECTL} ${KUBECTL_OPTS} apply -f - \
+    --namespace="${namespace}" -l ${ADDON_MANAGER_LABEL}=Reconcile 2>&1) && \
+      log INFO "== Successfully started ${config_name} in namespace ${namespace} at $(date -Is)" && \
```

Comment:
Would be great to log this slightly differently so that we can distinguish ensure and reconcile? e.g. `Successfully reconciled ...`